core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,23 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              var lastParameter = UrlService.getUrlParams();
+              if (lastParameter.parent) {
+                var parent = [];
+                parent.push(lastParameter.parent);
+                ExplorationRecommendationsService.getRecommendedSummaryDicts(
+                  parent,
```

Comment:
Replace this with `[parentExplorationId]`

Reply:
Done